transportable-udfs-codegen/src/main/java/com/linkedin/transport/codegen/CodegenUtils.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.codegen;
+
+import com.google.common.base.Preconditions;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collection;
+
+
+class CodegenUtils {
+
+  private CodegenUtils() {
+  }
+
+  static void writeCodeFile(Path outputDir, String packageName, String className, String extension,
```

Comment:
javadocs and clarify requirements for package-name. Is it required? outputDir must not exist or should be a dir